test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -53,41 +55,41 @@ class test_ucp_rma : public test_ucp_memheap {
     }
 
     void put_b(size_t size, void *target_ptr, ucp_rkey_h rkey,
-               void *expected_data, void *arg) {
+               void *expected_data, ucp_mem_h memh, void *arg) {
         ucs_status_ptr_t status_ptr = do_put(size, target_ptr, rkey,
-                                             expected_data, arg);
+                                             expected_data, memh, arg);
         request_wait(status_ptr);
     }
 
     void put_nbi(size_t size, void *target_ptr, ucp_rkey_h rkey,
-                 void *expected_data, void *arg) {
+                 void *expected_data, ucp_mem_h memh, void *arg) {
         ucs_status_ptr_t status_ptr = do_put(size, target_ptr, rkey,
-                                             expected_data, arg);
+                                             expected_data, memh, arg);
         request_release(status_ptr);
     }
 
     void put_nbi_iov(size_t size, void *target_ptr, ucp_rkey_h rkey,
-                     void *expected_data, void *arg) {
+                     void *expected_data, ucp_mem_h memh, void *arg) {
         do_nbi_iov(&test_ucp_rma::do_put_iov, size, target_ptr, rkey,
                    expected_data, arg);
     }
 
     void get_b(size_t size, void *target_ptr, ucp_rkey_h rkey,
-               void *expected_data, void *arg) {
+               void *expected_data, ucp_mem_h memh, void *arg) {
```

Comment:
i think memh should be after size (local side parameters are first)

Reply:
can change the order in the callback too..

Reply:
done, but requires updating many places